example/index.tsx

```diff
@@ -192,6 +193,17 @@ function MainForm({ subscription }: any) {
 		<KeyboardDatePicker label="Pick a date" name="date" required={required.date} dateFunsUtils={DateFnsUtils} />,
 		<DatePicker label="Birthday" name="birthday" required={required.birthday} dateFunsUtils={DateFnsUtils} />,
 		<TimePicker label="Break time" name="break" required={required.break} dateFunsUtils={DateFnsUtils} />,
+
+		<MuiPickersUtilsProvider utils={DateFnsUtils}>
+			<KeyboardDatePicker label="Pick a date (Provided Provider)" name="date" required={required.date} />
+		</MuiPickersUtilsProvider>,
+		<MuiPickersUtilsProvider utils={DateFnsUtils}>
+			<DatePicker label="Birthday (Provided Provider)" name="birthday" required={required.birthday} />
+		</MuiPickersUtilsProvider>,
+		<MuiPickersUtilsProvider utils={DateFnsUtils}>
+			<TimePicker label="Break time (Provided Provider)" name="break" required={required.break} />
+		</MuiPickersUtilsProvider>,
+
```

Comment:
Do we need to provide this three separate times? =) See silly stuff small copy/paste mistakes like this are exactly why doing it as a provider doesn't make sense. ;-)

Reply:
I only did this in the examples because this is an array and gets mapped over. otherwise all 3 get shoved into the single `Grid item`, though I can (and did do this originally) and it still looks fine?

Reply:
At least for the sake of examples its deceiving of the benefit I think? Any real world app using pickers will hoist the provider up above the form or even the entire app like a redux store or similar.

Reply:
Oh, I see... how about this then... why not hoist it up in this example too? Just wrap the higher level form.

Reply:
The example doesn't actually need to show every single way of doing things. I just wanted to make sure all the components are covered.